### PR TITLE
Create Stacks page

### PR DESCRIPTION
**Pull request by PaulFarault**
Create a "Stacks" page with the content of the [main TDP README.md](https://github.com/TOSIT-IO/TDP/blob/main/README.md).

This PR is for content approval. I'll add another one later to improve the display (using React components).

### CODE REVIEW

File: content/pages/1.discover/05.stacks/index.en.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+nav_title: Stacks
+---
+
+# Stacks TDP
+
+The following tables show the core components of TDP as well as the Apache branch they were based on, the `-basic` branch which serves as base for our work and the `-fix` branch which includes patches for the correct functionning.
```
Comment (wdavidw) [inline content/pages/1.discover/05.stacks/index.en.md:7]: We shall start by defining what is a stack, eg a collection of components with their associated version named after Ambari and Hortonworks deployment model.

**Comment by PaulFarault** [comment]
PR is now ready (except for @wdavidw 's comment). I needed to find a hack as the website was not able to parse MDX file where "too long" object are passed as component props. I solved this problem by introducing intermediary components (as `Stack31` for example).

**Comment by PaulFarault** [comment]
Also, we may want to move the "Stack Hadoop 3.3" part to another PR @rpignolet

**Comment by rpignolet** [comment]
> Also, we may want to move the "Stack Hadoop 3.3" part to another PR @rpignolet

You can remove the "Stack Hadoop 3.3" from this PR yes.

**Comment by PaulFarault** [comment]
I think that I'm now done with it. It only lacks a french translation if everybody agree with the new page intro. @rpignolet @Pierrotws @wdavidw

**Pull request merged**